examples/basic/versions.tf

```
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 0.13.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "> 4.8.0"
+    }
+    archive = {
+      source  = "hashicorp/archive"
+      version = "> 2.0.0"
```

```suggestion
      version = ">= 2.4.0"
```
Could we use the latest version?